### DISCUSSION TITLE
Estructura de directorios de fullstack

### DISCUSSION THREAD

**Discussion by probonogo** [probonogo/fullstackopen-Course-and-testing#59] [Q&A] [answered]
¿Dónde puedo encontrar ejemplos de la estructura de directorios de las respuestas?

**Comment by patchamama** [comment, marked as answer]
Los ejercicios se envían a través de GitHub y marcando los ejercicios completados en la [solicitud de envío](https://studies.cs.helsinki.fi/stats/courses/fullstackopen).

Puede enviar todos los ejercicios de este curso al mismo repositorio o utilizar varios repositorios. Si envía ejercicios de diferentes partes en el mismo repositorio, utilice un esquema de nomenclatura razonable para los directorios.

Una estructura de archivos muy funcional para el repositorio de envíos es la siguiente:

part0
part1
  courseinfo
  unicafe
  anecdotes
part2
  phonebook
  countriescopy
Vea [esto](https://github.com/fullstack-hy2020/example-submission-repository)!

Para cada parte del curso hay un directorio, que se ramifica en directorios que contienen una serie de ejercicios, como "unicafe" para la parte 1.

Por cada aplicación web para una serie de ejercicios, se recomienda enviar todos los archivos relacionados con ese aplicación, excepto para el directorio node_modules.

Los ejercicios se envían una parte a la vez. Cuando haya enviado los ejercicios para una parte del curso, ya no podrá enviar ejercicios incompletos para la misma parte.

Tenga en cuenta que en esta parte, hay más ejercicios además de los que se encuentran a continuación. No envíe su trabajo hasta que haya completado todos los ejercicios que desea enviar para la parte.
